From 05e3687c6c973c30bf35f3b7f4a7589b5030a830 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 31 Jul 2024 13:19:26 +0100
Subject: [PATCH 1203/1215] overlays: i2c-rtc: Correct bq32000 property name

The DT property for the BQ32000 controlled by trickle-resistor-ohms
parameter should be "trickle-resistor-ohms", not "abracon,tc-resistor".

See: https://github.com/raspberrypi/linux/issues/6291

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/i2c-rtc-common.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/i2c-rtc-common.dtsi
+++ b/arch/arm/boot/dts/overlays/i2c-rtc-common.dtsi
@@ -354,7 +354,7 @@
 					<&rv3028>,"trickle-resistor-ohms:0",
 					<&rv3032>,"trickle-resistor-ohms:0",
 					<&rv1805>,"abracon,tc-resistor:0",
-					<&bq32000>,"abracon,tc-resistor:0";
+					<&bq32000>,"trickle-resistor-ohms:0";
 		trickle-voltage-mv = <&rv3032>,"trickle-voltage-millivolts:0";
 		backup-switchover-mode = <&rv3028>,"backup-switchover-mode:0";
 		wakeup-source = <&ds1339>,"wakeup-source?",
